src/Microsoft.TemplateEngine.Cli/HelpFormatter.cs

```diff
@@ -238,35 +189,112 @@ int GetColumnWidth(int column)
             return b.ToString();
         }
 
+        private void CalculateColumnWidth(Dictionary<int, int> columnWidthLookup)
```

Comment:
minor: if you don't want to update the dictionary in this method, consider accepting a [readonlydictionary](https://docs.microsoft.com/en-us/dotnet/api/system.collections.objectmodel.readonlydictionary-2?view=netcore-3.1) instead.